demos/python_demos/image_retrieval_demo/image_retrieval_demo.py

```diff
@@ -58,6 +58,8 @@ def build_argparser():
                       help="Optional. Required for CPU custom layers. Absolute path to "
                            "a shared library with the kernels implementations.", type=str,
                       default=None)
+    args.add_argument('--no_show', action='store_true',
+                      help='Optional. Do not display output.')
```

Comment:
`CONTRIBUTING.md` claims it should be  `Optional. Do not visualize inference results.`